styles.css

```diff
@@ -0,0 +1,194 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+
+  /* background: #E5E5E5; */
+  font-family: 'Inter', sans-serif;
+}
+
+/* layout for smaller screens. breakpoint: 375px */
+.wrapper {
+  width: 375px;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: column;
+  height: 92px;
+  position: fixed;
+}
+
+/* header section */
+.statusbar,
+.logo-hamburger {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 44px;
+}
+
+.time {
+  position: absolute;
+  width: 54px;
+  height: 21px;
+  top: 14.5px;
+  left: 21px;
+}
+
+.bars {
+  position: absolute;
+  width: 66.66px;
+  height: 11.34px;
+  top: 17.33px;
+  left: 293.67px;
+  display: flex;
+}
+
+.bars img {
+  flex-grow: 1;
+}
+
+/* logo-hamburger section */
+.logo {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  text-decoration: none;
+  font-size: 24px;
+  font-weight: bold;
+  color: #172b4d;
+  left: 14px;
+  border: 1px solid #36b37e;
+  border-radius: 50%;
+  margin-top: 4px;
+}
+
+.hamburger {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  padding: 8px;
+  right: 24px;
+}
+
+/* section 2: headline */
+.headline {
+  position: absolute;
+  top: 92px;
+  width: 375px;
+  height: 100%;
+  background: url("./media/backgroundfill.png");
+  background-size: cover;
```

Comment:
:red_circle: I think the value here should be _contain_ to match the Figma Design. You can see below the how your design looks like after updating the value :arrow_down: 

Before
![before](https://github.com/tan12082001/Project-portfolio/assets/26736582/3f2a586b-f984-4651-9aba-45fdc1b6fc9e)


after

![after](https://github.com/tan12082001/Project-portfolio/assets/26736582/0a3a1973-efe1-4656-94a5-cf30feb244d3)


I think this second design is close to the Figma design. What do you think :thinking: